Modules/Filtering/DisplacementField/include/itkDisplacementFieldToBSplineImageFilter.hxx

```diff
@@ -34,7 +34,17 @@ DisplacementFieldToBSplineImageFilter<TInputImage, TInputPointSet, TOutputImage>
   DisplacementFieldToBSplineImageFilter()
 
 {
-  this->SetNumberOfRequiredInputs(0);
+
+  // Input 0 DisplacementField optional
+  Self::SetPrimaryInputName("DisplacementField");
+  Self::RemoveRequiredInputName("DisplacementField");
+  // Input 1 ConfidenceImage optional
+  Self::AddRequiredInputName("ConfidenceImage", 1);
```

Comment:
If a PointSet is provided instead of DisplacementField, will this still be required?

Reply:
None of the inputs are marked as required per the ProcessObject's book keeping. Manually in the VerifyPreconditions, it is checked that DisplacementField or PointSet is set.